aws_codeseeder/_cfn_seedkit.py

```diff
@@ -66,21 +69,23 @@ def synth(
     if permissions_boundary_arn:
         input_template["Resources"]["CodeBuildRole"]["Properties"]["PermissionsBoundary"] = permissions_boundary_arn
 
-    output_template = Template(yaml.dump(input_template, Dumper=yaml_dumper.get_dumper()))
+    role_prefix = kwargs.get("role_prefix", "/")
+    policy_prefix = kwargs.get("policy_prefix", "/")
 
-    LOGGER.debug("Writing %s", output_filename)
-    os.makedirs(out_dir, exist_ok=True)
-    account_id, _, partition = get_sts_info(session=session)
-    with open(output_filename, "w") as file:
-        file.write(
-            output_template.safe_substitute(
-                seedkit_name=seedkit_name,
-                account_id=account_id,
-                region=get_region(session=session),
-                partition=partition,
-                deploy_id=deploy_id,
-                role_prefix="/",
-            )
-        )
+    output_template = Template(yaml.dump(input_template, Dumper=yaml_dumper.get_dumper()))
+    template = output_template.safe_substitute(
+        seedkit_name=seedkit_name,
+        deploy_id=deploy_id,
+        role_prefix=role_prefix,
+        policy_prefix=policy_prefix,
+    )
 
-    return output_filename
+    if not synthesize:
+        LOGGER.debug("Writing %s", output_filename)
+        os.makedirs(out_dir, exist_ok=True)
+        with open(output_filename, "w") as file:
+            file.write(template)
+        return output_filename
+    else:
+        sys.stdout.write(template)
```

Comment:
Will this get mixed in with log output? Is that OK?

Reply:
Redirecting this into a file i.e:
```
codeseeder deploy seedkit test --synth > test.yaml
```
Doesn't mix it up with the log output and produces a valid template